client/source/App.jsx

```diff
@@ -28,4 +28,4 @@ const App = function CreateAppComponent() {
   );
 };
 
-export default App;
+export default App;
```

Comment:
EOF needs a new line. You probably just editing this file by mistake.

Reply:
Updated